packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -0,0 +1,51 @@
+@forward 'settings';
+
+@use 'usa-file-input/src/styles/usa-file-input';
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-hint/src/styles/usa-hint';
+@import '../../global/formation_overrides';
+
+/* Original Styles */
```

Comment:
This is a nitpick but with all of the other USWDS variations, we have added the USWDS classes to the top of the stylesheet above the `/* Original Styles */` comment.

I'm not sure why we're diverging in this component. For example: https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-radio/va-radio.scss